### DISCUSSION TITLE
Nested content in a table

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3344] [Q&A] [answered]
Hello! We would like to nest some content within a table row. My idea is to utilize the Disclosure component as a custom slot inside the table row. Could you please confirm if this is the correct approach? Additionally, do you have any recommendations on how we can achieve a similar effect more effectively?

<img width="992" alt="Zrzut ekranu 2023-07-24 o 15 26 55" src="https://github.com/twilio-labs/paste/assets/140410476/836d4210-441f-411e-8750-f4b3366f3bc2">

**Comment by serifluous** [comment, marked as answer]
Hey @marektonik ! Check out our reply in this discussion about why we don't support disclosures or expandable/collapsible sections inside Tables: #851 

This other discussion also has some examples of ways to use Disclosures, Tabs, and Tables to achieve what you might be looking for: #1153 + [Figma](https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?type=design&node-id=121-3602&mode=design&t=ZwKLEvMnqxRLqxtw-4)

Feel free to sign up for our office hours if you'd like more in-depth support on this specific use case! https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049

**Reply by marektonik** [reply]
Thanks @serifluous for information and links to other resources!

**Reply by charleneyjh1** [reply]
Hi @marektonik, I came across the same problem. Could you share what was your final approach on this? Thanks!

**Reply by marektonik** [reply]
@charleneyjh1 we decided to not use a table
[Figma](https://www.figma.com/file/oeUbF1qpn2FUsmXwiTsHmN/Segment-App%3A-Paste-Migration?type=design&node-id=2180-375713&mode=design&t=fLY3Sb6H4j7i27et-4)

<img width="1440" alt="Settings - User preferences - Manage Subscriptions - 02 (1)" src="https://github.com/twilio-labs/paste/assets/140410476/f116d8f6-bde1-46b0-a3f8-c534c2886e44">
